.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - run:
           name: 🏗️ Install dependencies (dash)
           command: |
-              git clone --depth 1 git@github.com:plotly/dash.git dash-main
+              git clone --depth 1 -b loosen-testing-reqs git@github.com:plotly/dash.git dash-main
```

Comment:
we can revert this part and merge this PR even before https://github.com/plotly/dash/pull/1506 merges, because all I'm doing here is adding more linting exclusions.